### ISSUE TITLE
Table mode for type displays functions without parentheses

### ISSUE THREAD

**Issue by Sytten** [typedoc2md/typedoc-plugin-markdown#696]
### What package is the bug related to?

typedoc-plugin-markdown

### Describe the issue

When using the `table` mode for `--typeDeclarationFormat`, it will display the functions without the parentheses which is very confusing visually since you have no way of knowing if it's a function or a property.
It is a bit annoying that those are considers properties in the first place (guessing this is a limitation of the compiler).

For example:
```markdown
# Type Alias: Response

> **Response**: `object`

An immutable saved Response.

## Type declaration

| Name | Type |
| ------ | ------ |
| `getBody` | `undefined` \| [`Body`](../classes/Body.md) |
| `getCode` | `number` |
| `getCreatedAt` | `Date` |
| `getHeader` | `undefined` \| `string`[] |
| `getHeaders` | [`Record`](Record.md)\<`string`, `string`[]\> |
| `getId` | [`ID`](ID.md) |
| `getRaw` | [`ResponseRaw`](ResponseRaw.md) |
| `getRoundtripTime` | `number` |
```

### TypeDoc configuration

`--typeDeclarationFormat=table`

### Expected behavior

Display the name with `(...args)`

**Comment by tgreyuk** [comment]
Thanks - agreed. This will go into next release;

**Comment by tgreyuk** [comment]
fix in fix in typedoc-plugin-markdown@4.2.9.
